chaoslib/openebs/jiva_controller_container_kill.yml

```diff
@@ -0,0 +1,80 @@
+- name: Derive PV from application PVC
+  shell: >
+    kubectl get pvc {{ app_pvc }} -n {{ app_ns }} --no-headers 
+    -o custom-columns=:spec.volumeName
+  args:
+    executable: /bin/bash
+  register: pv
```

Comment:
@w3aman  use failed_when condition to check the PV name can be obtained.

Reply:
Added !